pkg/lib/packagekit.js

```diff
@@ -301,13 +301,13 @@ export function cancellableTransaction(method, arglist, progress_cb, signalHandl
  * Returns: Icon classes; put them into <span class="returnvalue">&nbsp;</span>
  *
  */
-export function getSeverityIcon(info, secSeverity) {
+export function getSeverityIcon({ info, secSeverity, iconSize = "fa-sm" }) {
```

Comment:
I was thinking about it. But since the API is so far used only in 1 file, I thought it might be more effective to change the API.

Reply:
Right, it's not *hard* to change the API -- I'm just saying that it is very unusual to call a function with a single object as parameter instead of just regular positional parameters.

Reply:
It was just my preference, because secServerity is also an optional argument, which would result in calling the function like `getSeverityIcon(info, undefined, "fa-md")` I know that's totally fine in JS, but I just didn't like how it looked visually, passing "undefined" parameter.

Reply:
Ah, I see. Sad lack of named function parameters in JS (:heart: Python).